### ISSUE TITLE
Electron 7.0 breaks Riot Desktop 1.5.0-1 on Arch Linux

### ISSUE THREAD

**Issue by emilysnow2477** [vector-im/riot-web#11272]
Expected behavior: GUI opens, can send messages and interact with the client as normal.
Actual behavior: the GUI doesn't open, when `riot-desktop` is run from the command line it displays this error message:

`(node:3) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed`

Without looking into it further, it suggests that some part of the code is hanging due to using a deprecated method that electron no longer supports. This breakage will likely happen on most rolling release systems, like Void, Gentoo, Arch, etc.

**Comment by turt2live** [comment]
We don't currently support Electron 7 - please try Electron 6 instead.

**Comment by neVERberleRfellerER** [comment]
Riot 1.5.0 works just fine (well, it complains that all E2EE devices are unverified, but that might be unrelated) for me on Arch with Electron 7. Make sure that it's not minimized in systray, which is what happens to me (it starts minimized).

**Comment by dkasak** [comment]
Yeah, it works fine for me too. I'm not experiencing the E2EE verification visual glitch but I have some visual glitches with seen markers. They sometimes appear to start moving and then snap back into the previous position and freeze there for a bit or they overlap each other.

**Comment by emilysnow2477** [comment]
It [mostly] works for me at this point. For whatever reason, I upgraded one day and it just started working again. Doesn't make a lot of sense. I did uninstall and reinstall `firejail`, which sandboxes my programs, so that could be possibly related. But I tried running `firejail --noprofile [executable]` and it didn't make any difference, as well as me trying to uninstall + reinstall `firejail` earlier on to no avail, so that explanation doesn't seem entirely founded to me.

I've experienced some similar visual glitches to @dkasak, however. In any case, Riot's more or less reverted back to a workable stage.
